sql/2019/17_CDN/17_08.sql

```diff
@@ -0,0 +1,9 @@
+#standardSQL
+# 17_08: Percentage of responses with keep-alive header
+SELECT
+  _TABLE_SUFFIX AS client,
+  ROUND(SUM(IF(LOWER(respOtherHeaders) LIKE "%keep-alive%", 1, 0)) / COUNT(0), 4) AS pctKeepAlive

```

Comment:
Query the `resp_keep_alive` field directly.

Reply:
Changed to resp_keep_alive  != ' ' and COUNTIF